### PR TITLE
Identity release archboard feedback

### PR DESCRIPTION
**Pull request by christothes**

### CODE REVIEW

File: sdk/identity/Azure.Identity/CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## 1.10.0-beta.2 (Unreleased)
 
 ### Features Added
-- Add `BrowserCustomizedOptions` to `InteractiveBrowserCredential` to enable web view customization for interactive authentication.
+- Added `BrowserCustomizations` property to `InteractiveBrowserCredential` to enable web view customization for interactive authentication.
```
Comment (schaabs) [inline sdk/identity/Azure.Identity/CHANGELOG.md:6]: This is plural, but the actual API is singular `BrowserCustomization`. Personally, I like the plural `BrowserCustomizations` better, but the singular was what was suggested in the API review.

File: sdk/identity/Azure.Identity/src/Credentials/InteractiveBrowserCredentialOptions.cs [hunk 2]
```diff
@@ -69,6 +69,6 @@ public string TenantId
         /// <summary>
         /// The options for customizing the browser for interactive authentication.
         /// </summary>
-        public BrowserCustomizationOptions BrowserCustomizedOptions { get; set; }
+        public BrowserCustomizationOptions BrowserCustomization { get; set; }
```
Comment (schaabs) [inline sdk/identity/Azure.Identity/src/Credentials/InteractiveBrowserCredentialOptions.cs:72]: Should this be `BrowserCustomizations`?

Comment (christothes) [reply sdk/identity/Azure.Identity/src/Credentials/InteractiveBrowserCredentialOptions.cs:72]: I think the singular is what was intended and was the naming agreed upon, so I think I'll keep it this way.

File: sdk/identity/Azure.Identity/CHANGELOG.md (hunk 1 above)
Comment (christothes) [reply sdk/identity/Azure.Identity/CHANGELOG.md]: Sticking with the singular, so I'll change it here.

File: sdk/identity/Azure.Identity/tests/InteractiveBrowserCredentialTests.cs [hunk 3]
```diff
@@ -294,25 +294,25 @@ public async Task InvokesBeforeBuildClientOnExtendedOptions()
         }
 
         [Test]
-        public async Task BrowserCustomizedOptionsHtmlMessage([Values(null, "<p> Login Successfully.</p>")] string htmlMessageSuccess, [Values(null, "<p> An error occured: {0}. Details {1}</p>")] string htmlMessageError)
+        public async Task BrowserCustomizationsHtmlMessage([Values(null, "<p> Login Successfully.</p>")] string htmlMessageSuccess, [Values(null, "<p> An error occured: {0}. Details {1}</p>")] string htmlMessageError)
```
Comment (scottaddie) [inline sdk/identity/Azure.Identity/tests/InteractiveBrowserCredentialTests.cs:297]: Nit: Should "Customizations" be singular instead?

Comment (christothes) [reply sdk/identity/Azure.Identity/tests/InteractiveBrowserCredentialTests.cs:297]: Probably, but shouldn't matter for a test.

**Review by schaabs — APPROVED** [review]

**Pull request merged**